src/Utilities/CssConcatenator/RuleBlock.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pelago\Emogrifier\Utilities\CssConcatenator;
+
+/**
+ * Implementation of media rules sub-property for \Pelago\Emogrifier\Utilities\CssConcatenator
+ *
+ * @internal
+ *
+ * @author Jake Hotson <jake.github@qzdesign.co.uk>
+ * @author SignpostMarv
+ */
+class RuleBlock
+{
+    /**
+     * Array whose keys are selectors for the rule block (values are of no
+     *     significance);
+     *
+     * @var mixed[]
+     *
+     * @psalm-var array<string, mixed>
+     */
+    public $selectorsAsKeys = [];
+
+    /**
+     * The property declarations, e.g. "margin-top: 0.5em; padding: 0".
+     *
+     * @var string
+     */
+    public $declarationsBlock;
+
+    /**
+     * @param mixed[] $selectorsAsKeys
+     * @param string $declarationsBlock
```

Comment:
This docblock tag appears to be redundant, being needed solely to satisfy phpcs?

Reply:
I already replied to this.  GitHub/Firefox going wrong again.  It's needed for tools like PhpDocumentor.

Reply:
the thinking I'm going with here is that a scalar param docblock with no description adds no value that can't already be extracted through reflection/AST

Reply:
It also shows that the author of the code has thought about it (and upon reflection realized that their naming of the method/parameter is so good that there is nothing more to add in the DocBlock).  For that reason, I think having a skeletal DocBlock is important, and it should have all ribs.

Reply:
A missing DocBlock looks like the programmer forgot to put it in at all.